cirq-google/cirq_google/engine/qcs_notebook.py

```diff
@@ -152,3 +134,68 @@ def get_qcs_objects_for_notebook(
         processor_id=processor_id,
         is_simulator=is_simulator,
     )
+
+
+def get_hardware_engine_and_authenticate_user(
```

Comment:
The long name makes this function harder to reason about and calling it as just as long as calling the constituent parts. WYDT of exposing `authenticate_user` and calling that from the notebook instead?

Reply:
+1, updated to make `authenticate_user` public